spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployer.java

```diff
@@ -190,45 +195,51 @@ public void shutdown() throws Exception {
 		}
 	}
 
+	/**
+	 * Will check if {@link LocalDeployerProperties#INHERIT_LOGGING} is set by
+	 * checking deployment properties and then application properties
+	 */
+	private boolean shouldInheritLogging(AppDeploymentRequest request){
+		boolean logToConsole = false;
+		if (request.getDeploymentProperties().containsKey(LocalDeployerProperties.INHERIT_LOGGING)){
+			logToConsole = Boolean.parseBoolean(request.getDeploymentProperties().get(LocalDeployerProperties.INHERIT_LOGGING));
+		}
+		else if (request.getDefinition().getProperties().containsKey(LocalDeployerProperties.INHERIT_LOGGING)){
```

Comment:
This is a deployer only concern IMHO, and so should not be checked in the definition